docs/README.rst

```diff
@@ -0,0 +1,217 @@
+Readit
+======
+
+|Python| |Licence| |Build Status| |docs passing|
+
+
+.. image:: https://asciinema.org/a/XHY1hKrgOry69OeWG04iucT1k.png
+   :alt: asciicast
+   :target: https://asciinema.org/a/XHY1hKrgOry69OeWG04iucT1k?t=1
+
+
+
+Introduction
+============
+Readit is command line bookmark manager. It is a command line utility to add, delete, update and display the bookmarks. It is a powerful bookmark manager written in Python. It uses SQLite3 database to store the bookmarks.
+
+
+
+.. contents:: 
```

Comment:
We already have `index.rst` which has index of all sections also we have index on left side so It will be better if we remove this one.